extra/logging_config.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+version: 1
+disable_existing_loggers: False
+formatters:
+  standard:
+    format: '%(asctime)s [%(levelname)s] %(name)s - %(message)s'
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: INFO
+    formatter: standard
+    stream: ext://sys.stdout
+  file:
+    class: logging.FileHandler
+    level: INFO
+    formatter: standard
+    filename: /data/vusualizer/VUSualizer/extra/output.log
+root:
+  level: DEBUG
+  handlers: [console, file]
```

Comment:
Based on #8, i expected `email` here as well.

Reply:
I did remove "email" in the handlers, because I did not use it. I just finished some small tests by adding the handler to act on "logging.error", but I cannot get it to work properly right now.
I already made an git issue #11 , based on another pull request. I think we use a different way to send emails within our team, so I rather want to have this implemented in a future release.